src/careamics/lvae_training/calibration.py

```diff
@@ -0,0 +1,178 @@
+from typing import Union
+
+import numpy as np
+import torch
+from scipy import stats
+
+
+def get_last_index(bin_count, quantile):
+    cumsum = np.cumsum(bin_count)
+    normalized_cumsum = cumsum / cumsum[-1]
+    for i in range(1, len(normalized_cumsum)):
+        if normalized_cumsum[-i] < quantile:
+            return i - 1
+    return None
+
+
+def get_first_index(bin_count, quantile):
+    cumsum = np.cumsum(bin_count)
+    normalized_cumsum = cumsum / cumsum[-1]
+    for i in range(len(normalized_cumsum)):
+        if normalized_cumsum[i] > quantile:
+            return i
+    return None
+
+
+class Calibration:
```

Comment:
Can you add a quick description of what the calibration is used for?

Reply:
Sure thing